packages/flutter/lib/src/widgets/animated_child_switcher.dart

```diff
@@ -10,39 +10,148 @@ import 'framework.dart';
 import 'ticker_provider.dart';
 import 'transitions.dart';
 
-class _AnimatedChildSwitcherChildEntry {
-  _AnimatedChildSwitcherChildEntry(this.widget, this.controller, this.animation);
+/// Represents a child that is, or was previously set on the
```

Comment:
you either need an extra comma after "previously" or to remove the comma after "is", otherwise the sentence becomes "{Represents a child that is} or {was previously set on the [AnimatedChildSwitcher.child] field that needs to be laid out using a [AnimatedChildSwitcherLayoutBuilder].}", which makes no sense.

Reply:
Fixed.

Reply:
wait, i'm confused. I thought this was the interface you had to pass to the transition builder, but actually you only use it for the layout builder. Why not just pass the layout builder a `List<Widget>`?